Essentials/src/messages.properties

```diff
@@ -207,6 +207,7 @@ inventoryClearingAllStack=\u00a76Cleared all\u00a7c {0} \u00a76from {1}\u00a76.
 inventoryClearingFromAll=\u00a76Clearing the inventory of all users...
 inventoryClearingStack=\u00a76Removed\u00a7c {0} \u00a76of\u00a7c {1} \u00a76from {2}\u00a76.
 is=is
+realName={0} is {1}
```

Comment:
This should be alphabetically placed, like the other related strings.

Reply:
Okey, Give me a minute to change.